src/tensor_ops.jl

```diff
@@ -256,6 +256,23 @@ end
 Base.ctranspose(S::AllTensors) = transpose(S)
 
 
+############################
+# Symmetric/Skew-symmetric #
+############################
+
+"""
+Computes the symmetric part of a second order tensor.
+"""
+@inline sym{dim,T}(S1::Tensor{2,dim,T}) = convert(SymmetricTensor{2,dim,T},S1)
```

Comment:
But... Same thing here. It is really strange that a symmetric function would not return a symmetric tensor. I basically want this to be an easier way to convert to a symmetric tensor. Lets say I want to use the output for a double contraction. Then I want it to be a symmetric tensor type since the double product is faster for symmetric tensors. If I would use sym which returns a tensor, and then convert it after, then the convert operation does the same calculations again.

I dunno, just makes sense to me that `sym` would return a symmetric tensor :)